src/System.Net.Primitives/src/System/Net/CredentialCache.cs

```diff
@@ -144,13 +145,6 @@ public void Remove(string host, int port, string authenticationType)
             _cacheForHosts.Remove(key);
         }
 
-        /// <devdoc>
```

Comment:
Found the discussion about this: #230. No strong feelings about these: it sounds like we're going to have them extracted from the doc database at a later point.